.github/workflows/main.yml

```diff
@@ -634,6 +634,11 @@ jobs:
       - name: Install dependencies (npm ci)
         run: npm ci --prefer-offline
 
+        # Check to see if we should build webviews or not
+      - name: Check if we need to build webviews
```

Comment:
I think this makes what is happening a bit more obvious. I went with a not if check as the default for most of the notebook checks is to skip webview build. Also previously only the 'notebook' step was skipping this. Now the other steps that don't need this (remoteNotebook, nonRawNotebook...) will also skip it for the perf gain.